tests/infra/logging_app.py

```diff
@@ -2,61 +2,173 @@
 # Licensed under the Apache 2.0 License.
 
 import infra.checker
+import suite.test_requirements as reqs
+import time
 
 from loguru import logger as LOG
 
 
+@reqs.description("Running transactions against logging app")
+@reqs.supports_methods("LOG_record", "LOG_record_pub", "LOG_get", "LOG_get_pub")
+def test_run_txs(
+    network,
+    args,
+    logging_app,
+    nodes=None,
+    num_txs=1,
+    verify=True,
+    timeout=3,
+    ignore_failures=False,
+    wait_for_sync=False,
+):
+    if nodes is None:
+        nodes = network.get_joined_nodes()
+    num_nodes = len(nodes)
+
+    for tx in range(num_txs):
+        logging_app.issue_on_node(
+            network=network,
+            remote_node=nodes[tx % num_nodes],
+            number_txs=1,
```

Comment:
`number_txs=num_txs`?

Reply:
this sends `num_txs` transactions in total, it just sends 1 to each node in a round robin fashion